### PR TITLE
Fix retransmitter file name

### LINKED ISSUE #1122: typo in retransmitter filename
I just noticed the extra `i` in `src/source/PeerConnection/Retransimitter.c` :)
vs `src/source/PeerConnection/Retransmitter.h`

### PR DESCRIPTION
**Pull request by disa6302**
*Issue #, if available:*

*Description of changes:*

Resolves #1122 
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by hassanctech — APPROVED** [review]

**Comment by zhuker** [comment]
fixes https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/issues/1122

**Comment by zhuker** [comment]
it'd be awesome if this also included fix for the ifdef in `Retransmitter.h`
```
#ifndef __KINESIS_VIDEO_WEBRTC_CLIENT_PEERCONNECTION_RETRANSIMITTER__
```
note the extra `I` in `_RETRANSIMITTER__`

File: src/source/PeerConnection/Retransmitter.h [hunk 1]
```diff
@@ -1,8 +1,8 @@
 /*******************************************
 Retransmitter internal include file
 *******************************************/
-#ifndef __KINESIS_VIDEO_WEBRTC_CLIENT_PEERCONNECTION_RETRANSIMITTER__
-#define __KINESIS_VIDEO_WEBRTC_CLIENT_PEERCONNECTION_RETRANSIMITTER__
+#ifndef __KINESIS_VIDEO_WEBRTC_CLIENT_PEERCONNECTION_RETRANSMITTER__
+#define __KINESIS_VIDEO_WEBRTC_CLIENT_PEERCONNECTION_RETRANSMITTER__
```
Comment (hassanctech) [inline src/source/PeerConnection/Retransmitter.h:5]: #endif LN 27 also needs this fix

Comment (disa6302) [reply src/source/PeerConnection/Retransmitter.h:5]: Had fixed it. Change was not pushed :(

**Pull request merged**